src/OrchardCore.Modules/OrchardCore.Users/Controllers/AccountController.cs

```diff
@@ -179,5 +205,136 @@ private IActionResult RedirectToLocal(string returnUrl)
                 return Redirect("~/");
             }
         }
+
+        [HttpGet]
+        [AllowAnonymous]
+        public async Task<IActionResult> ForgotPassword()
+        {
+            if (!(await _siteService.GetSiteSettingsAsync()).As<RegistrationSettings>().EnableLostPassword)
+            {
+                return NotFound();
+            }
+
+            return View();
+        }
+
+        [HttpPost]
+        [AllowAnonymous]
+        public async Task<IActionResult> ForgotPassword(ForgotPasswordViewModel model)
+        {
+            if (ModelState.IsValid)
```

Comment:
You can send email without checking
`_siteService.GetSiteSettingsAsync()).As<RegistrationSettings>().EnableLostPassword` via for instance Postman.

Reply:
TODO